Jenkinsfile

```diff
@@ -60,7 +60,7 @@ node('mongodb-2.4') {
     }
 
     stage("Set up content schema dependency") {
-      govuk.contentSchemaDependency()
+      govuk.contentSchemaDependency(env.SCHEMA_BRANCH)
```

Comment:
This looks good, but I discovered the other day that you can use `params.SCHEMA_BRANCH` instead. That means you can also delete the `initializeParameters` call on line 37.